src/Compilers/CSharp/Portable/Symbols/Source/SourcePropertySymbol.cs

```diff
@@ -13,6 +13,8 @@ namespace Microsoft.CodeAnalysis.CSharp.Symbols
 {
     internal sealed class SourcePropertySymbol : SourcePropertySymbolBase
     {
+        private SourcePropertySymbol? _otherPartOfPartial;
```

Comment:
We'll want to keep an eye on perf numbers and make sure that this doesn't hurt memory consumption much. It's nowhere near as bad as adding a new field to SyntaxNode, but property symbols aren't uncommon.

Reply:
I see, the concern is this is not strictly pay-for-play since it's increasing the size of a fairly common symbol type. Sure, let's make sure to test insert before merging the feature, and if memory issues arise we could break out a further subtype.